packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -13,12 +13,21 @@ import '../cache.dart';
 import '../dart/package_map.dart';
 import '../globals.dart';
 import '../runner/flutter_command.dart';
+import '../test/coverage_collector.dart';
 import '../test/flutter_platform.dart' as loader;
 import '../toolchain.dart';
 
 class TestCommand extends FlutterCommand {
   TestCommand() {
     usesPubOption();
+    argParser.addFlag('coverage',
+      defaultsTo: false,
+      help: 'Whether to collect coverage information.'
+    );
+    argParser.addOption('coverage-path',
+      defaultsTo: 'coverage/lcov.info',
```

Comment:
if it's relative to the current working directory, then having a directory as part of the default seems weird.